index.js

```diff
@@ -58,19 +57,25 @@ class MultiStream extends stream.Readable {
     this._forwarding = false
   }
 
-  destroy (err) {
```

Comment:
We should not override `destroy` as that will bypass important teardown logic and event ordering.